tck/features/expressions/pattern/Pattern1.feature

```diff
@@ -38,7 +38,7 @@ Feature: Pattern1 - Pattern predicate
       """
     When executing query:
       """
-      MATCH (n) WHERE (n)-[]->() RETURN n
+      MATCH (n) WHERE NOT isEmpty((n)-[]->()) RETURN n
```

Comment:
More or less this entire file was testing patterns which were coerced to boolean. So perhaps we want to remove it fully instead or change the last few tests which are supposed to fail to also use isEmpty?

Reply:
I think the tests still serve a purpose. However, I agree that the negative tests in the end of the file should also be updated to include `isEmpty`.

Reply:
No. I think all your changes on naked pattern in `WHERE` go beyond what was decided, far as I am aware of. Naked pattern where not banned completely. They are still allowed in contexts there clearly a boolean is expected and the evaluate to a boolean in these context.

This [CIP2015-05-13 EXISTS](https://github.com/openCypher/openCypher/blob/master/cip/1.accepted/CIP2015-05-13-EXISTS.adoc#212-syntax-rules) summarizes this (the link goes directly to the crucial section). The was written to document the decision made by in the CLG and was also reviewed by the CLG down the line, as far as I remember things.

Reply:
Hmm, the description in this card for the deprecations https://trello.com/c/AjC65NYS/121-deprecate-coercion-of-lists-to-boolean agrees with you. But what I did was to hack Neo4j to throw errors instead of warnings on deprecations and check which TCK tests failed. And all usages here looked like they was deprecated in the code. I will investigate a bit more...